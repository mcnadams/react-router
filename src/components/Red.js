import React from 'react';

export default function Red() {
  const style = {
    background: 'red',
    width: '250px',
    height: '250px'
  };

  return <div style={style}></div>;
}
