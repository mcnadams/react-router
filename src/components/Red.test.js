import React from 'react';
import { shallow } from 'enzyme';
import Red from './Red';

describe('Red', () => {
  it('Red', () => {
    const wrapper = shallow(<Red />);
    expect(wrapper).toMatchSnapshot();
  });
});
