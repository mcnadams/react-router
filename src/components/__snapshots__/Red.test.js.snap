// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Red Red 1`] = `
<div
  style={
    Object {
      "background": "red",
      "height": "250px",
      "width": "250px",
    }
  }
/>
`;
